src/components/Hand/Hand.js

```diff
@@ -2,13 +2,70 @@ import React from 'react';
 import handBg from './HandBg.svg';
 import handFg from './HandFg.svg';
 import './Hand.css';
+import PropTypes from 'prop-types';
 
-const Hand = props => (
-  <div className="hand">
-    <img className="hand-component" src={handBg} alt="" />
-    <img className="hand-snack" src={props.snack} alt="" />
-    <img className="hand-component" src={handFg} alt="" />
-  </div>
-);
+const Hand = props => {
+  const itemsToRotate = [
+    'boost',
+    'crunchie',
+    'curly-wurly',
+    'fruit-pastilles-roll',
+    'getbuzzing-mint-choc-high-protein',
+    'getbuzzing-mixed-berries',
+    'kit-kat-chunky',
+    'mars',
+    'mentos',
+    'misc-bar',
+    'nakd-apple-crunch',
+    'nakd-berry-delight',
+    'nakd-cashew-cookie',
+    'nakd-cocoa-orange',
+    'nakd-coconut-bliss-nibbles',
+    'nature-valley-maple-syrup',
+    'nature-valley-oats-dark-chocolate',
+    'nature-valley-oats-n-honey',
+    'panda-liquorice',
+    'skittles',
+    'smarties',
+    'snickers',
+    'trek-banana-bread-flapjack',
+    'trek-peanut-power',
+    'twix',
+    'winegums',
+    'wrigleys'
+  ];
+  let rotate = false;
+  for (const i in itemsToRotate) {
+    if (props.snack.includes(itemsToRotate[i])) {
+      rotate = true;
+      break;
+    }
+  }
+
+  const freddo = props.snack.includes('freddo');
```

Comment:
There's a convention for prefixing boolean variables with `is`, `has` etc which can help identify them immediately

As discussed offline, it might be good to have a list of "weird" items that require bespoke styling

e.g.

```
const longItems = ['freddo'] // There's probably a better name for them!
const isLongItem = longItems.some(longItem => longItem.includes(props.snack)) 
```